packages/grid/x-grid-modules/src/hooks/features/useSelection.ts

```diff
@@ -110,10 +111,12 @@ export const useSelection = (
 
   const selectRows = React.useCallback(
```

Comment:
Have you considered using `useEventCallback` instead of useCallback?
It will remove the need for the new `optionsRef`.

Reply:
yes we can do if you want

Reply:
No strong preferences. If you find it easier in the future or solving bugs, I think that it will be great to consider it.